src/components/sectioningConfirm/ConfirmLabware.tsx

```diff
@@ -25,19 +25,31 @@ import { LayoutPlan } from "../../lib/machines/layout/layoutContext";
 import { CommentFieldsFragment, ConfirmSectionLabware } from "../../types/sdk";
 import { selectConfirmOperationLabware } from "./index";
 import RemoveButton from "../buttons/RemoveButton";
+import { ConfirmationModal } from "../modal/ConfirmationModal";
 
 interface ConfirmLabwareProps {
   originalLayoutPlan: LayoutPlan;
   comments: Array<CommentFieldsFragment>;
   onChange: (labware: ConfirmSectionLabware) => void;
-  onRemoveClick: (labwareBarcode: string) => void;
+  onSectionUpdate: (layoutPlan: LayoutPlan) => void;
+  onSectionNumberChange: (
+    layoutPlan: LayoutPlan,
+    slotAddress: string,
+    sectionIndex: number,
+    sectionNumber: number
+  ) => void;
+  removePlan: (barcode: string) => void;
+  autoMode: boolean;
```

Comment:
How come the mode here is represented as a boolean instead of your SectionNumberMode enum?

Reply:
Updated to enum